user-access-service/src/test/java/no/unit/nva/database/IdentityServiceTest.java

```diff
@@ -357,11 +368,42 @@ void createTableThrowsExceptionWhenDynamoClientIsNull() {
     void createMapperOverridingHardCodedTableNameLogsErrorSayingThatMapperIsNull() {
         TestAppender appender = LogUtils.getTestingAppender(IdentityServiceImpl.class);
         Executable action =
-            () -> createTable(null, mockEnvironment());
+            () -> createTable(null);
         assertThrows(RuntimeException.class, action);
         assertThat(appender.getMessages(), containsString(DYNAMO_DB_CLIENT_NOT_SET_ERROR));
     }
 
+    @Test
+    void shouldReturnPageOfResultsWhenScanDatabaseRequestIsSubmitted()
```

Comment:
This is the test that tests the scanning function. The whole functionality is tested by the handler so this is mostly for test-coverage. Since though we are writing tests, I need to add one more that exhibits that the whole table can be scanned.